### PR TITLE
(ayekan) Configure Prometheus for Mimir remote write and deploy Mimir

### PR DESCRIPTION
**Pull request by fbegyn**
This deploys the Mimir cluster to Ayekan and ensures that the local prometheus writes to it. It also configures the prometheus to only use serviceMonitors from `o11y.eu/monitor=enabled` namespaces.

### CODE REVIEW

File: ayekan/prometheus/prometheus.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#! /usr/bin/env bash
```
Comment (jhoblitt) [inline ayekan/prometheus/prometheus.sh:1]: The space after the `#!` is much less common than no space.

**Review by jhoblitt — CHANGES_REQUESTED** [review]
This looks good other than than the small nitpick about the shebang.

**Comment by fbegyn** [comment]
Fixed the shebang nit

File: ayekan/prometheus/prometheus.sh (hunk 1 above)
Comment (fbegyn) [reply ayekan/prometheus/prometheus.sh:1]: fixed it in the last commit

**Review by jhoblitt — APPROVED** [review]
LGTM.

**Pull request merged**